appinventor/blocklyeditor/src/exportBlocksImage.js

```diff
@@ -461,11 +464,11 @@ Blockly.exportBlockAsPng = function(block) {
     img.src = uri;
     img.onload = function() {
       var canvas = document.createElement('canvas');
-      canvas.width = img.width;
-      canvas.height = img.height;
+      canvas.width = 2*(img.width);
```

Comment:
Parentheses are not necessary here. Code style: space around operators.

Reply:
Fixed